packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -1491,15 +1493,45 @@ const useEcharts = (
                         return '';
                     })
                     .join('');
-                const tooltipHeader =
-                    params[0].dimensionNames?.[0] !== undefined
-                        ? getFormattedValue(
-                              params[0].axisValueLabel,
-                              params[0].dimensionNames[0],
-                              items,
-                          )
-                        : params[0].axisValueLabel;
-                return `${tooltipHeader}<br/><table>${tooltipRows}</table>`;
+
+                const dimensionId = params[0].dimensionNames?.[0];
+
+                if (dimensionId !== undefined) {
+                    const field = items.find(
+                        (item) => getItemId(item) === dimensionId,
+                    );
+
+                    if (
+                        isDimension(field) &&
+                        (field.type === DimensionType.DATE ||
+                            field.type === DimensionType.TIMESTAMP)
+                    ) {
+                        const date = (params[0].data as Record<string, any>)[
+                            dimensionId
+                        ]; // get full timestamp from data
+                        const dateFormatted = formatDate(
```

Comment:
All timestamps lose the time information.  Could we use `getFormattedValue` ? otherwise `formatTimestamp` 

<img width="877" alt="Screenshot 2023-10-12 at 09 29 42" src="https://github.com/lightdash/lightdash/assets/9117144/9d0a85d9-bfb5-4801-9bb5-ecc2b89f0f65">

Reply:
Yep, done 

![image](https://github.com/lightdash/lightdash/assets/1983672/89dc1e1f-bfa0-40b3-ad10-07c47bb105ab)